ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/DockerStatic/Dockerfiles/Dockerfile.ubi8

```diff
@@ -0,0 +1,36 @@
+FROM redhat/ubi8
+# Install Base Requirements
+RUN dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
```

Comment:
`RUN dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm http://mirror.centos.org/centos/8-stream/BaseOS/x86_64/os/Packages/centos-gpg-keys-8-2.el8.noarch.rpm http://mirror.centos.org/centos/8-stream/BaseOS/x86_64/os/Packages/centos-stream-repos-8-2.el8.noarch.rpm`

To be able to installl xorg-x11-server-Xvfb, I had to install the centos8 stream repo, and its keys, this way

Reply:
I've made some similar changes to the dockerfile in the latest commit :)

Reply:
Oh sorry, I didnt see those. I'll test them now 👍🏻